core/crypto/crypto_store/crypto_store_impl.cpp

```diff
@@ -74,39 +95,19 @@ namespace kagome::crypto {
     return outcome::success();
   }
 
-  outcome::result<std::string> CryptoStoreImpl::loadFile(
-      const Path &file_path) const {
-    if (!boost::filesystem::exists(file_path)) {
-      return CryptoStoreError::FILE_DOESNT_EXIST;
-    }
-
-    std::ifstream file;
-    auto close_file = gsl::finally([&file] {
-      if (file.is_open()) file.close();
-    });
-
-    file.open(file_path.string(), std::ios::in);
-    if (!file.is_open()) {
-      return CryptoStoreError::FAILED_OPEN_FILE;
-    }
-
-    std::string content;
-    file >> content;
-    return content;
-  }
-
   CryptoStoreImpl::CryptoStoreImpl(
       std::shared_ptr<ED25519Provider> ed25519_provider,
       std::shared_ptr<SR25519Provider> sr25519_provider,
       std::shared_ptr<Secp256k1Provider> secp256k1_provider,
       std::shared_ptr<Bip39Provider> bip39_provider,
       std::shared_ptr<CSPRNG> random_generator)
-      : keys_directory_(),
+      : keys_directory_{},
```

Comment:
This is redundant. Default constructor will be called defaultly, we don't need to call that specially.

Reply:
Any comments?

Reply:
You might be seeing an old commit, I already removed it few commits ago.